doc/manual/50-feature-details.en.md

```diff
@@ -539,6 +539,21 @@ Retention policy based on recovery window
     backups required to allow point-in-time recovery back to 9:30 AM
     on the previous Friday.
 
+> **IMPORTANT:**
+> Incremental backups are ignored during the retention policy execution. This
+> due to the fact that an `incremental backup` is only valid if the
+> entire chain of backups is intact. So, the only backup considered
+> for retention policy is the `root backup` (also called `full backup`).
+>
+> How does it work?
+>
+> When the retention policy is enforced, barman will ignore the
+> incremental backups and check for the status of the
+> `root backup`. If the status is `KEEP:FULL` or `KEEP:STANDALONE`,
+> the status `VALID` will be propagated to the whole chain of children
+> backups, otherwise the current status of the `root backup` will be
+> propagated.
+
```

Comment:
Yeah, i think it looks good! 

I asked to my friend chat-gepeto to rewrite your suggestion to be as clear as possible to an end user and it gave me this:

Incremental backups are not considered during the retention policy process. This is because incremental backups depend on either a full backup or other incremental backups to be valid. To maintain the consistency of backup chains, only full backups are reviewed when applying the retention policy.

How It Works
When the retention policy is applied, Barman will ignore incremental backups and focus on the status of the full backup, also known as the "root backup" if there are incremental backups dependent on it.

If the full backup is marked as KEEP:FULL, KEEP:STANDALONE, or VALID, this status is extended to all dependent incremental backups.
If the full backup is marked as OBSOLETE, then all incremental backups that depend on it will also be marked as OBSOLETE and removed.

Reply:
> If the full backup is marked as KEEP:FULL, KEEP:STANDALONE, or VALID, this status is extended to all dependent incremental backups.
> If the full backup is marked as OBSOLETE, then all incremental backups that depend on it will also be marked as OBSOLETE and removed

I don't think this part is 100% correct. I mean, the `keep` status is not propagated to the children, but rather they are considered valid.

I guess I would keep only the second paragraph, i.e. explain that if a full backup is marked as obsolete, so is the whole tree of children backups.

Reply:
I have made a modification mixing up suggestions! I think we could maintain everything so we are certain that things are explained in details but i have change the parts where @barthisrael mentioned that are not 100% correct. Please, review it again when possible!